Banks/creation-banks/creation.css

```
@@ -0,0 +1,239 @@
+    position: absolute;
+    top: 8px;
+    margin-left: -28px;
+    transform: translate(-50%, -50%);
+    font-size: 29px;
+    color: green;
+}
+
+.second-title {
+    position: absolute;
+    top: 8px;
+    margin-left: 18px;
+    transform: translate(-50%, -50%);
+    font-size: 29px;
+    color: blue;
+}
+
+.profile {
+    width: 85px;
+    height: 85px;
+    border-radius: 50%;
+    overflow: hidden;
+    margin-top: -570px;
+    margin-left: 37px;
+}
+
+.profile img {
+    width: 85px;
+    height: 85px;
+    margin-top: 1px;
+    margin-left: 1px;
+    object-fit: cover;
+    object-position: center;
+}
+
+.name {
+    align-items: center;
+    position: absolute;
+    margin-top: 10px;
+    margin-left: 25px;
+    font-size: 22px;
+    color: gray;
+}
+
+.bancos {
+    align-items: center;
+    color: gray;
+    font-size: 23px;
+    margin-top: 70px;
+    margin-left: 50px;
+}
+
+.second-container {
+    align-items: center;
+    margin-top: -48px;
+    margin-left: 18px;
+}
+
+.agência {
+    color: gray;
+    font-size: 23px;
+    margin-top: 39px;
+    margin-left: 49px;
+}
+
+.firts-container {
+    align-items: center;
+    margin-top: -49px;
+    margin-left: 17px;
+}
+
+.conta {
+    color: gray;
+    align-items: center;
+    font-size: 23px;
+    margin-left: 47px;
+    margin-top: 35px;
+}
+
+.third-container {
+    align-items: center;
+    margin-top: -48px;
+    margin-left: 19px;
+}
+
+.customers {
+    color: gray;
+    align-items: center;
+    font-size: 23px;
+    margin-left: 47px;
+    margin-top: 38px;
+}
+
+.four-container {
+    align-items: center;
+    margin-top: -48px;
+    margin-left: 15px;
+}
+
+.sub-container {
+    background: white;
+    margin-top: -440px;
+    width: 1296px;
+    border: 1px solid white;
+    padding: 30px;
+}
+
+.navegation-button {
+    margin-left: 190px;
+    height: 45px;
+    padding-left: 40px;
+    border-radius: 10px;
+    border: 1px solid gray;
+}
+
+.five-container {
+    margin-left: 200px;
+    margin-top: -36px;
+}
+
+.sub-title {
+    font-size: 47px;
+    margin-top: 30px;
+    margin-left: 570px;
+}
+
+.name-complete {
+    color: gray;
+    margin-top: -71px;
+    margin-left: 264px;
+    font-size: 15px;
+}
+
+.button {
+    position: relative;
+    height: 40px;
+    width: 460px;
+    border-radius: 8px;
+    border: 1px solid gray;
+    top: -10px;
+    left: 260px;
+}
+
+.date {
+    margin-bottom: 10px;
+    color: gray;
+    font-size: 15px;
+    margin-top: 13px;
+    margin-left: 775px;
+}
+
+.button-bank {
+    position: relative;
+    height: 40px;
+    width: 460px;
+    border-radius: 8px;
+    border: 1px solid gray;
+    top: -5px;
+    left: 770px;
+}
+
+.phone-number {
+    color: gray;
+    margin-top: 12px;
+    margin-left: 773px;
+    font-size: 15px;
+}
+
+.button-phone {
+    height: 35px;
+    width: 460px;
+    margin-top: -50px;
+    margin-left: 770px;
+    border-radius: 8px;
+    border: 1px solid gray;
+}
+
+.Endereço {
```

Tirar acento, nunca usar acento ou Ç, nenhuma linguagem suporta. Prefira usar o inglês, não correrá esse risco. Algumas linguagens são Case-sensitive, elas diferenciam maiúsculas e minúsculas, prefira usar as letras minúsculas para classes em css. Lembrando, ao trocar o nome aqui, trocar em TODOS os locais que chamar
```suggestion
.endereco {
```